src/pages/test-report/cat-d/components/pcv-door-exercise/__tests__/pcv-door-exercise.spec.ts

```diff
@@ -0,0 +1,721 @@
+  //     component.hasSeriousFault = true;
+  //     component.isDangerousMode = true;
+  //
+  //     const storeDispatchSpy = spyOn(store$, 'dispatch');
+  //     component.addOrRemoveFault(true);
+  //
+  //     expect(storeDispatchSpy).not.toHaveBeenCalledWith(new AddDangerousFault(component.competency));
+  //   });
+  // });
+  //
+  // describe('competencyHasFault', () => {
+  //   it('should return true if competency has a driving fault', () => {
+  //     component.faultCount = 1;
+  //
+  //     expect(component.competencyHasFault()).toBe(true);
+  //   });
+  //   it('should return true if competency has a serious fault', () => {
+  //     component.hasSeriousFault = true;
+  //
+  //     expect(component.competencyHasFault()).toBe(true);
+  //   });
+  //   it('should return true if competency has a dangerous fault', () => {
+  //     component.hasDangerousFault = true;
+  //
+  //     expect(component.competencyHasFault()).toBe(true);
+  //   });
+  //   it('should return false if competency does not have a fault', () => {
+  //     component.faultCount = undefined;
+  //     component.hasDangerousFault = false;
+  //     component.hasSeriousFault = false;
+  //
+  //     expect(component.competencyHasFault()).toBe(false);
+  //   });
+  // });
+  //
+  // describe('canAddSingleFault', () => {
+  //   it('should return false if competency has an existing fault (faultCount >= 1)', () => {
+  //     component.faultCount = 1;
+  //     expect(component.canAddSingleDrivingFault()).toBe(false);
+  //   });
+  //   it('should return true if faultCount is undefined and competency was pressed', () => {
+  //     component.faultCount = undefined;
+  //     expect(component.canAddSingleDrivingFault(true)).toBe(true);
+  //   });
+  //   it('should return false if faultCount is undefined and competency was not pressed', () => {
+  //     component.faultCount = undefined;
+  //     expect(component.canAddSingleDrivingFault()).toBe(false);
+  //   });
+  // });
+  //
+  // describe('shouldDisableRippleForOneFaultLimit', () => {
+  //   it('should return false if more than one fault permitted', () => {
+  //     component.faultCount = 2;
+  //     component.oneFaultLimit = false;
+  //
+  //     expect(component.shouldDisableRippleForOneFaultLimit()).toBe(false);
+  //   });
+  //   it('should return true if only one fault permitted', () => {
+  //     component.faultCount = 1;
+  //     component.oneFaultLimit = true;
+  //
+  //     expect(component.shouldDisableRippleForOneFaultLimit()).toBe(true);
+  //   });
+  // });
+  //
+  // describe('hasDrivingFault', () => {
+  //   it('should return true if faultCount is not undefined', () => {
+  //     component.faultCount = 1;
+  //     expect(component.hasDrivingFault()).toBe(true);
+  //   });
+  //   it('should return false if faultCount is undefined', () => {
+  //     component.faultCount = undefined;
+  //     expect(component.hasDrivingFault()).toBe(false);
+  //   });
+  // });
+});
```

Comment:
Can we comment out all tests - they're not relevant to pcv-door-exercise so better to comment out until tests are dealt with.